### PR TITLE
Community Mentorship Program Newsarticle

### PR DESCRIPTION
**Pull request by Okorin**
Disclaimer: Ephemeral told me to do this if I want a newspost for this, if this is completely wrong please no yelling

### CODE REVIEW

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: restore to tumblr link except use 163075746433 as the id

see previous news-articles for example

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: rename file to all lower-case and add a proper date placeholder

we'll go with 17th, so

2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
+date: 2017-07-xx xx:xx:xx +0000
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: change to day 17 whatever time (24hrs) it is when you get around to changing this in UTC

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+tumblr_url: no idea what to do with these fields
+---
+
+Have you ever thought: “Wow, I’m bad at mapping”? Does the quote “So let’s be honest, you suck at mapping” (pishifat, 2016)  strike a personal chord with you? Then the Community Mentorship Program is the place for you!
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: Have you ever thought, "wow, I'm bad at mapping"? Does pishifat's "you suck at mapping" quote strike a personal chord? The Community Mentorship Program is the place for you, read on to find out more!

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/ZXakpTkXEC0" frameborder="0" allowfullscreen></iframe>
+
+[seems to need a banner of sorts?]
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: banner goes before iframe content, you'll need to make one or have one made for this. check previous newsposts for the particular dimensions, we have a "something" newsposts template using scorewatch/osuweekly motifs (but non coloured) we can use if you haven't got anything

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles each lasting approximately 10 weeks. Aspiring mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
+
+
+What is the Community Mentorship Program?
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: this is well written and i have no qualms with it, but it's a lot of words for a newsposts article that is essentially introducing the mentor program and advertising that applications are open for it. maybe have this in the thread you link to, and shorten the post down as a whole to basically:

* this is what the program does and what activities it offers
* this is who it's intended for
* this is how to apply
* this is when dates end for applying
* go fuckin apply

also nobody cares about the catch integrations as that involves exactly nobody not already in the program. newsposts are general/all reach, so try and not have specific content referenced that most people will not care/do not know about

File: news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+The contests encourage mentees to put in their best effort and perhaps experiment with new things to gain that competitive edge. Mentors are allowed to guide their mentees in a general sense but are prohibited from directly changing the map.
+
+osu!catch Integration
+-----
+Lastly, the osu!catch mentorship discord will now be merging with the osu!standard mentorship discord, and community members from both modes will be part of the same server. There is a possibility of integrating the osu!mania and osu!taiko mentorship programs as well, but nothing has been finalized as of yet.
+
+For more details, refer to the [forum post for cycle #5](https://osu.ppy.sh/forum/p/6132973)!
+
+— Helpers and Organizers of the Community Mentorship Program
```
Comment (Ephemeralis) [inline news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md]: replace Helpers with Okorin, this is important for old web display

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 20:45:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: /wiki/ not wiki/

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 20:45:00 +0000
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: pop this over to today's date as well

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: title case now -> Now

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles, each lasting approximately 10 weeks. Aspiring new mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
+
+The mentorship program is a place where newer mappers can seek out personalized assistance with developing their mapping knowledge. Activities include mapping contests, discussions, and playtesting each other’s maps for feedback.
+
+If you want to apply - either as a mentor or mentee - fill in the application forms found on the forum post for cycle #5! Mentor applications are open until July 28th and mentee applications will run from July 30th to August 11th.
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: link to the post in any text that mentions it

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 20:45:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles, each lasting approximately 10 weeks. Aspiring new mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: add a little tl;dr after this with something like

"If you're interested in learning to mod and map from some of osu!'s best and brightest, this is the place for you!"

**Comment by Okorin** [comment]
okay, im now about 70% sure that i did changes accordingly

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](/wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles, each lasting approximately 10 weeks. Aspiring new mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
+
+This program is a place where newer mappers can seek out personalized assistance with developing their mapping knowledge. Aside from just mapping chit-chat the server holds other activities such as internal mapping contests for mentees, mapping lectures in a sort of classroom setting as well as playtesting each other’s maps for feedback.
+
+Did this spark your interest? If you want to apply - either as a mentor or mentee - read the [forum post for cycle #5](https://osu.ppy.sh/forum/p/6132973) and fill in the corresponding application forms found on there!
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: on there ->> within

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](/wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles, each lasting approximately 10 weeks. Aspiring new mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
+
+This program is a place where newer mappers can seek out personalized assistance with developing their mapping knowledge. Aside from just mapping chit-chat the server holds other activities such as internal mapping contests for mentees, mapping lectures in a sort of classroom setting as well as playtesting each other’s maps for feedback.
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: comma after chit-chat

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups Now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 22:15:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](/wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: drop Community Mentorship Program caption from the image link

File: news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md [hunk 16]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups Now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 22:15:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
```
Comment (Ephemeralis) [inline news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md]: "in mapping and modding"

may -> might

Add "Read on to find out more and how to apply!"

**Comment by Okorin** [comment]
i mean - the banner is just like a thing we made because we needed a banner for this post
i personally dont care because we dont even have a banner by default

only one that could get slightly upset is Naxess, but that's just my problem aint it

**Comment by Ephemeralis** [comment]
It's not a bad banner, just have to make sure it's within the incredibly nebulous not-written-down-anywhere-or-possibly-even-existing-at-all design guidelines for frontpage content. Better safe than sorry, etc.

**Comment by Ephemeralis** [comment]
Looks like we're good on the banners!

I'll give this one last pass tomorrow after I wake up and get it out. Will tweet about it as well.

**Pull request merged**